lib/tasks/temporary/hacktoberfest_badges.rake

```diff
@@ -0,0 +1,34 @@
+# TODO: Will be removed my @mstruve once badges are rewarded for 2020
+# rubocop:disable  Metrics/BlockLength
+task award_hacktoberfest_badges: :environment do
+  badge_id = Badge.find_by(slug: "hacktoberfest-2020")&.id
+  user_ids = UserSubscription.where(
+    user_subscription_sourceable_id: 529_344,
```

Comment:
is this this the id for `https://dev.to/devteam/how-to-claim-your-hacktoberfest-completion-badge-3l2h`?

Reply:
yes